griptape/tools/base_tool.py

```diff
@@ -121,6 +122,7 @@ def execute(self, activity: Callable, subtask: ActionsSubtask, action: ToolActio
     def before_run(self, activity: Callable, subtask: ActionsSubtask, action: ToolAction) -> Optional[dict]:
         return action.input
 
+    @observable(tags=["Tool.run()"])
```

Comment:
Same question here, why the tag?

Reply:
Same answer as above. It is so that we can group all spans for tools together. We can then use that to answer questions like, "how much time was spent running tools?".